### PR TITLE
Release/2.0.2

### PR DESCRIPTION
**Pull request by bunnyc1986**

### CODE REVIEW

File: lib/square_connect/api_client.rb [hunk 1]
```diff
@@ -27,12 +24,13 @@ class ApiClient
     # @return [Hash]
     attr_accessor :default_headers
 
+    # Initializes the ApiClient
+    # @option config [Configuration] Configuration for initializing the object, default to Configuration.default
     def initialize(config = Configuration.default)
       @config = config
-      @user_agent = "square-connect-ruby/#{VERSION}"
```
Comment (chriskillpack) [inline lib/square_connect/api_client.rb]: Was this change to `@user_agent` intentional?

File: spec/api/location_api_spec.rb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    end
+  end
+
+  # unit tests for list_locations
+  # ListLocations
+  # Provides the details for all of a business&#39;s locations.  Most other Connect API endpoints have a required &#x60;location_id&#x60; path parameter. The &#x60;id&#x60; field of the [&#x60;Location&#x60;](#type-location) objects returned by this endpoint correspond to that &#x60;location_id&#x60; parameter.
+  # @param authorization The value to provide in the Authorization header of your request. This value should follow the format &#x60;Bearer YOUR_ACCESS_TOKEN_HERE&#x60;.
+  # @param [Hash] opts the optional parameters
+  # @return [ListLocationsResponse]
+  describe 'list_locations test' do
```
Comment (chriskillpack) [inline spec/api/location_api_spec.rb]: 👍

File: lib/square_connect/api_client.rb (hunk 1 above)
Comment (chriskillpack) [inline lib/square_connect/api_client.rb]: Let's make sure that this change is not something we made intentionally in our original fork of the SDK templates. IIRC we made some change around user agents.

Comment (bunnyc1986) [inline lib/square_connect/api_client.rb]: user-agent is not added back.

File: README.md [hunk 3]
```diff
@@ -1,15 +1,19 @@
-Connect v2 SDKs
+Connect v2 Ruby SDKs [![Build Status](https://travis-ci.org/square/connect-ruby-sdk.svg?branch=master)](https://travis-ci.org/square/connect-ruby-sdk)[![Gem Version](https://badge.fury.io/rb/square_connect.svg)](https://badge.fury.io/rb/square_connect)
```
Comment (chriskillpack) [inline README.md]: 👍

File: README.md [hunk 4]
```diff
@@ -22,6 +26,142 @@ Or add this line to your Gemfile:
 gem 'square_connect'
 ```
 
+### Install from Git
+
+If the Ruby gem is hosted at a git repository: https://github.com/GIT_USER_ID/GIT_REPO_ID, then add the following in the Gemfile:
```
Comment (chriskillpack) [inline README.md]: Is there anyway we can replace GIT_USER_ID and GIT_REPO_ID with our own info?

File: docs/SortOrder.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# SquareConnect::SortOrder
```
Comment (chriskillpack) [inline docs/SortOrder.md]: Not a blocker but strange to have an empty file here. We should investigate the cause of these, maybe related to the orphaned models we need to clean up from `api.json`.

Comment (bunnyc1986) [inline docs/SortOrder.md]: Yes. it's worth to investigate. Th python doc has its content: https://github.com/square/connect-python-sdk/blob/release/2.0.2/docs/SortOrder.md

File: README.md (hunk 4 above)
Comment (bunnyc1986) [inline README.md]: I think it's from the template. Will update.

**Comment by bunnyc1986** [comment]
the git info are added.

**Pull request merged**